python/src/tasks/mitochondrial_content.py

```diff
@@ -18,24 +19,32 @@ def _format_result(self, result):
         return [Result(result)]
 
     def compute(self):
-        # the cells to get MT-content data for
-        cells = self.task_def["cells"]
-
-        request = {"cells": cells}
+        # Retrieve the MitochondrialContent of all the cells
+        request = {}
         r = requests.post(
             f"{config.R_WORKER_URL}/v0/getMitochondrialContent",
             headers={"content-type": "application/json"},
             data=json.dumps(request),
         )
-        
-        resultR = r.json()
-        result = {}
-        for i in range(len(resultR['cells_id'])):
-            cell = resultR['cells_id'][i]
-            doublet_scores = resultR['percent.mt'][i]
 
-            result[cell] = {
-                "percent-MT": doublet_scores,
-            }
+        # Format result with the same structure than Gene Expression. The values are ordered by the default order
+        # of the Embedding in the Seurat object
+        # {"MT-content": {'min': , X 'max': , X 'mean': , X 'stdev': , X 'values': [...]}
+
+        result = {}
+        values = r.json()
+        minimum = float(np.amin(values))
+        maximum = float(np.amax(values))
+        mean = float(np.mean(values))
+        stdev = float(np.std(values))
+
+        result["MT-content"] = {
```

Comment:
For this one, you don't need to put the result as a dictionary under key, so you can just do `result = { ... }`